app/admin/routes.js

```diff
@@ -5,9 +5,9 @@ import AdminDashboardNavigate from './components/AdminDashboardNavigate';
 import Navbar from './components/NavBar';
 
 const Routes = () => (
-  <div>
+  <div className="Main">
     <Navbar />
-    <div>
+    <div className="Admin">
```

Comment:
What are these classnames used for?

Reply:
Nothing as of now. In case we need some custom styling in future.